projects/plugins/jetpack/modules/sso.php

```diff
@@ -206,6 +296,38 @@ public function create_error_notice_and_redirect( $query_params ) {
 		return wp_safe_redirect( $url );
 	}
 
+	/**
+	 * Adds 'Revoke invite' link to user table row actions.
+	 * Removes 'Reset password' link.
+	 *
+	 * @param array   $actions - User row actions.
+	 * @param WP_User $user_object - User object.
+	 */
+	public function jetpack_user_table_row_actions( $actions, $user_object ) {
+		$user_id            = $user_object->ID;
+		$has_pending_invite = self::has_pending_wpcom_invite( $user_id );
+
+		if ( current_user_can( 'promote_users' ) && $has_pending_invite ) {
+			$nonce                        = wp_create_nonce( 'jetpack-sso-revoke-user-invite' );
+			$actions['sso_revoke_invite'] = sprintf(
+				'<a class="jetpack-sso-revoke-invite-action" href="%s">%s</a>',
+				add_query_arg(
+					array(
+						'action'              => 'jetpack_revoke_invite_user_to_wpcom',
+						'user_id'             => $user_id,
+						'revoke_invite_nonce' => $nonce,
```

Comment:
Let's include the invite slug here instead of refetching it again in the handler.
```suggestion
						'revoke_invite_nonce' => $nonce,
						'invite_id'           => $has_pending_invite,
```

Reply:
Great optimization! I'll commit these suggestions.